projects/demo/src/modules/services/breakpoint/examples/inject-service.md

```diff
@@ -0,0 +1,13 @@
+```ts
+import {Inject, Self} from '@angular/core';
+import {TuiBreakpointService} from '@taiga-ui/core';
+
+// ...
+export class MyComponent {
+  constructor(
+    @Self()
```

Comment:
do we need add eslint rule in `@taiga-ui/eslint-plugin` for provide `@Self` as the same `tui-destroy-service`?

Reply:
oh sorry I will remove @Self